R/timeseriessummary-data.R

```diff
@@ -44,7 +44,14 @@ parseCustomDataElementsForTemplateForTimeSeriesSummary <- function(reportData) {
   
   approvalsTable <- formatDataTable(parseTSSApprovals(reportData, timezone))
   
+  tsDetailsTable <- list()
+  tsDetails <- constructTSDetails(reportData, timezone)
+  tsDetailsTable[['tsAttrs']] <- formatDataTable(tsDetails[['tsAttrs']])
+  tsDetailsTable[['tsExtAttrs']] <- formatDataTable(tsDetails[['tsExtAttrs']])
+  addChangeNote <- tsDetails[['changeNote']]
+  
   return(list(
+      tsDetails = list(hasData=TRUE, data=tsDetailsTable, addChangeNote=addChangeNote),
```

Comment:
In this case Larua wanted the section to always be open-able even when there is no data